### ISSUE TITLE
Accessibility - Web - Poor Color Contrast on the Step 8 Page

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#114]
When you open the second hide/show for English, Spanish, and French, the orange text in the gray box does not have enough color contrast. The color contrast ratio is 1.88:1 with the gray background and needs to be 4.5:1 or higher to pass.

Please see the screenshot below.

![complychain - spanish - step 8 page - poor color contrast](https://user-images.githubusercontent.com/10423487/43531298-7aed6da0-957d-11e8-8cc5-594579927a8c.png)

**Comment by sanganalakshay** [comment]
![image](https://user-images.githubusercontent.com/15368456/43806879-e4997604-9a73-11e8-9294-b18f261e93b6.png)


I checked on webaim site and the it shows me a ratio of 4.60.1.

Can you tell me the hex # of the gray background?

**Comment by jfc3-dol** [comment]
The gray background is: **#E0E0E0** and has a color contrast ratio of **3.9:1** with that orange.

**Comment by jfc3-dol** [comment]
I re-checked the color contrast using IE 11, Firefox, and Chrome and it now passes color contrast checker with a color contrast ratio of 4.6:1 which passes because it's higher than the required 4.5:1.

Closing issue.
